src/SFA.DAS.EmployerIncentives.Database/Tables/ApprenticeshipIncentive.sql

```diff
@@ -11,7 +11,8 @@
 	[PlannedStartDate] DATETIME2 NOT NULL,
 	[IncentiveApplicationApprenticeshipId] UNIQUEIDENTIFIER NOT NULL,
 	[AccountLegalEntityId] BIGINT NULL, 
-    [UKPRN] BIGINT NULL
+    [UKPRN] BIGINT NULL, 
+    [Withdrawn] BIT NOT NULL DEFAULT 0
```

Comment:
Wrong table.

Reply:
Withdrawals are at an apprenticeship level, so the Withdrawn flag should be on the IncentiveApplicationApprenticeship table